src/vs/workbench/contrib/notebook/browser/contrib/chatEdit/notebookChatEditController.ts

```diff
@@ -86,7 +86,7 @@ class NotebookChatEditorController extends Disposable {
 			if (!model || !session) {
 				return;
 			}
-			return session.entries.read(r).find(e => isEqual(e.modifiedURI, model.uri));
+			return session.entries.read(r).find(e => isEqual(e.modifiedURI, model.uri) && e.kind === 'text') as IModifiedTextFileEntry | undefined;
```

Comment:
Notebooks today only deal with the textual entries.
Once we bake in full support for notebooks, this will go away (this is part of the work around code implemented a a week ago)